rachelekm/vegishphilly
# Add Github Actions CI

## Overview

This PR adds support for Github Actions and runs `cibuild` as part of CI workflow. Workflow is triggered on push event and does not include support for deployment/publishing images to ECR.

Connects #14 

## Testing Instructions

 * Push a dummy commit to this branch and confirm that it triggers a build
 * Confirm that the CI workflow passes
